waspc/src/Wasp/Util.hs

```diff
@@ -67,12 +68,22 @@ headSafe :: [a] -> Maybe a
 headSafe [] = Nothing
 headSafe xs = Just (head xs)
 
+second :: (b -> d) -> (a, b, c) -> (a, d, c)
+second f (x, y, z) = (x, f y, z)
```

Comment:
`second` from `Control.Arrow` only works on pairs (and I have a tuple of 3). Is implementing something like this a code smell?